README.md

```diff
@@ -154,6 +154,33 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Pushing Changes to a Remote Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to a remote Cluster for others to review.  This involves building a local image, pushing the image to Quay, then updating a remote cluster to pull the new image.
```

Comment:
these lines are too long. please add som eline breaks :)

Reply:
When I look at the raw file, it displays like
![image](https://user-images.githubusercontent.com/12733153/58564001-268b9600-81fa-11e9-9874-f9612c36903f.png)
Does seem to wrap ok?

Reply:
It will render fine, but it makes it harder to look at inside an editor due to the long lines.